### PR TITLE
K-6: Find the icon for our organisation. Update README.md.

### LINKED ISSUE #6: Find the icon for our organisation and set it
Definition of done:
- a new icon is displayed in the organisation project

### PR DESCRIPTION
**Pull request by Boryszs**

### CODE REVIEW

**Review by Corel-Cormen — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -6,4 +6,6 @@ We want to create an archive with job offers
 
 ## Why are we doing this?
 
-We want to exchange knowledge and learn new things in our free time
\ No newline at end of file
+We want to exchange knowledge and learn new things in our free time
+
+![](https://cdn-icons-png.flaticon.com/256/1488/1488581.png)
```
Comment (mateuszjanczak) [inline README.md]: What if the icon is removed from this website? Please add it to repository

File: README.md [hunk 2]
```diff
@@ -6,4 +6,6 @@ We want to create an archive with job offers
 
 ## Why are we doing this?
 
-We want to exchange knowledge and learn new things in our free time
\ No newline at end of file
+We want to exchange knowledge and learn new things in our free time
+
+![](icon.png)
```
Comment (mateuszjanczak) [inline README.md]: Could you move the icon to the beginning and center it?

Comment (Boryszs) [reply README.md]: No problem my friend  ;)

**Review by mateuszjanczak — APPROVED** [review]
LGTM

**Pull request merged**